config/hscConfig.py

```diff
@@ -0,0 +1,19 @@
+# Configure to force using astrometry_net instead of pre-loaded reference catalogs
```

Comment:
"pre-loaded" doesn't seem quite the distinction.

Perhaps be very specific and say: "instead of the Pan-STARRS1 reference catalog."